easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.2.2-GCCcore-8.3.0.eb

```
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'Xerces-C++'
+version = '3.2.2'
+
+homepage = 'http://xerces.apache.org/xerces-c/'
+
+description = """Xerces-C++ is a validating XML parser written in a portable
+subset of C++. Xerces-C++ makes it easy to give your application the ability to
+read and write XML data. A shared library is provided for parsing, generating,
+manipulating, and validating XML documents using the DOM, SAX, and SAX2
+APIs."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+source_urls = ['http://archive.apache.org/dist/xerces/c/%(version_major)s/sources/']
+sources = ['xerces-c-%(version)s.tar.gz']
+checksums = ['dd6191f8aa256d3b4686b64b0544eea2b450d98b4254996ffdfe630e0c610413']
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+    ('binutils', '2.32'),
+]
+
+dependencies = [
+    ('cURL', '7.66.0'),
+]
+
```

Please add the test step as in `Xerces-C++-3.2.2-GCCcore-8.2.0.eb`
```suggestion

runtest = 'test'

```

---

tests added